### DISCUSSION TITLE
Children databases - AutoOpen

### DISCUSSION THREAD

**Discussion by T4ngml** [Kunzisoft/KeePassDX#1526] [Q&A]
Hi,
I just installed KeePassDX, and was wondering if it offers support for multiple databases, and especially children databases attached with the AutoOpen folder.

I didn't find how to do this in the app. Did I miss it?

Thanks!

**Comment by J-Jamet** [comment]
Not yet : https://github.com/Kunzisoft/KeePassDX/issues/209
